src/api/stakeholders.js

```diff
@@ -0,0 +1,196 @@
+import { handleResponse, handleError, getOrganizationId } from './apiUtils';
+import { session } from '../models/auth';
+const log = require('loglevel').getLogger('../api/stakeholders');
+
+const STAKEHOLDER_API = process.env.REACT_APP_STAKEHOLDER_API_ROOT;
+
+async function fetchJSON(query, options) {
+  return fetch(query, options).then(handleResponse).catch(handleError);
+}
+
+export default {
+  getStakeholders(id, { offset, rowsPerPage, orderBy, order, filter }) {
+    const orgId = id || getOrganizationId();
+    const filterObj = {
+      where: filter,
+      order: [`${orderBy} ${order}`],
+      limit: rowsPerPage,
+      offset,
+    };
+
+    let query = `${STAKEHOLDER_API}`;
+
+    if (orgId) {
+      query += `/${orgId}?filter=${JSON.stringify(filterObj)}`;
+    } else {
+      query += `?filter=${JSON.stringify(filterObj)}`;
+    }
+
+    log.debug('getStakeholders', query);
+
+    const options = {
+      method: 'GET',
+      headers: {
+        'content-type': 'application/json',
+        Authorization: session.token,
+      },
+    };
+
+    return fetchJSON(query, options);
+  },
+
+  getUnlinkedStakeholders(id, abortController) {
+    const orgId = getOrganizationId();
+    let query = `${STAKEHOLDER_API}/relations`;
+
+    if (id && orgId && orgId !== id) {
+      query += `/${id}?isRelation=${false}&owner_id=${orgId}`;
```

Comment:
Case in API parameters should be consistent. I'd expect `isRelation` to be `is_relation`.